src/usb.c

```diff
@@ -737,7 +737,7 @@ stlink_t *stlink_open_usb(enum ugly_loglevel verbose, bool reset, char serial[16
     }
 
     libusb_device **list;
-    int cnt = libusb_get_device_list(slu->libusb_ctx, &list);
+    int cnt = (int) libusb_get_device_list(slu->libusb_ctx, &list);
```

Comment:
We should use `ssize_t` and use it as a counter if > 0. As per libusb API:
`ssize_t 	libusb_get_device_list (libusb_context *ctx, libusb_device ***list)`